app/gym/constants.py

```diff
@@ -0,0 +1,41 @@
+"""
+Choices of repetition units for gym.set model
+"""
+REPETITIONS = 'RE'
```

Comment:
next step here will be to change these in an enum, but we will talk about that another time